src/Sylius/Bundle/FlowBundle/composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=5.5.9",
 
-        "symfony/framework-bundle": "~2.7.5"
+        "symfony/framework-bundle": "~2.7,>=2.7.5"
```

Comment:
We can simply use `^` instead of `~` and that should be corrected in all bundles...